src/hooks/useSubaccount.tsx

```diff
@@ -66,7 +66,7 @@ export const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: Lo
   const { usdcDenom, usdcDecimals } = useTokenConfigs();
   const { compositeClient, faucetClient } = useDydxClient();
 
-  const { getFaucetFunds } = useMemo(
+  const { getFaucetFunds, getNativeTokens } = useMemo(
```

Comment:
[q] is this related to the gas denom toggle functionality? not seeing this used anywhere else

Reply:
this is called in `requestFaucetFunds` / when you "desposit" on testnet via faucet